prismic/connection.py

```diff
@@ -66,7 +66,7 @@ def get_json(url, params=None, access_token=None, cache=None, ttl=None, request_
 
 
 def get_max_age(headers):
-    expire_header = headers["Cache-Control"]
```

Comment:
Ah, drats, I missed this PR--- I also fixed it over here.  https://github.com/prismicio/python-kit/issues/24